File: LeetCode/292.Nim_Game/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test
s = Solution()
if (s.canWinNim(1) == True):
    print("Pass")
else:
    print("Failed")
if (s.canWinNim(2) == True):
    print("Pass")
else:
    print("Failed")

File: LeetCode/292.Nim_Game/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    if (s.canWinNim(1) == true) { cout << "Test 1 - PASS" << endl;}
    else { cout << "Test 1 - Failed" << endl; }
    if (s.canWinNim(2) == true) { cout << "Test 2 - PASS" << endl;}
    else { cout << "Test 2 - Failed" << endl; }
    return 0;
}